spec/Vehicle/Vehicle.vspec

```diff
@@ -295,3 +295,123 @@
   type: attribute
   description: The CO2 emissions in g/km.
   unit: g/km
+
+
+#
+# Vehicle Weight and Dimension attributes
+#
+
+- CurrentOverallWeight:
+  datatype: uint16
+  type: sensor
+  value: 0
+  unit: kg
+  description: Current overall Vehicle weight, in kg, including all liquids and full tank of fuel.
```

Comment:
I know that we just moved this item, but is the description mentioning "full tank of fuel" correct? I.e. that the sensor providing this value needs to consider how full the tank is and adjust the value accordingly?

Reply:
My feeling is that "CurrentOverallWeight" is sort of the "best guess" how heavy a vehicle currently is, including everything potentially even cargo and passengers. (how accurate it is/can be depends on the vehicle).
@chheis may correct me 😀

Reply:
I follow @erikbosch. from the description I'd rather think it's a static value, but the type `sensor` indicates, that it's dynamic. Maybe we should clarify that.

Reply:
I changed the description according to the discussion